src/Nethermind/Nethermind.Baseline/ContractMerkleTree.cs

```diff
@@ -0,0 +1,13 @@
+using Nethermind.Abi;
+
+namespace Nethermind.Baseline
+{
+    public static class ContractMerkleTree
+    {
+        public static readonly AbiSignature InsertLeafAbiSig = new AbiSignature("insertLeaf",
```

Comment:
I suggest using AbiDefinition and parsing json ABI like in Contracts in AuRa

Reply:
yeah, I need a guidance on it